### PR TITLE
Devops role for mitogen install

### PR DESCRIPTION
**Pull request by tziebro**
Created role to install mitogen on CentOS7 and AmazonLinux2 instances. The role installs mitogen through pip with a umask of 0022 which allows all users to use mitogen.

### CODE REVIEW

File: mitogen/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+
+- name: add EPEL repo on CentOS
+  yum:
+    name: epel-release
+    state: present
+  when: ansible_distribution == 'CentOS' and ansible_distribution_major_version == '7'
+  tags: [ mitogen, epel ]
+
+- name: ensure pip is installed
```
Comment (matthewgaulin) [inline mitogen/tasks/main.yml]: Would using the pip ansible-role as a dependency make sense?

Comment (tziebro) [reply mitogen/tasks/main.yml]: > Would using the pip ansible-role as a dependency make sense?
Thanks for explaining this...I will make this a dependency.

File: mitogen/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+
+- name: add EPEL repo on CentOS
+  yum:
+    name: epel-release
+    state: present
+  when: ansible_distribution == 'CentOS' and ansible_distribution_major_version == '7'
```
Comment (tequalsme) [inline mitogen/tasks/main.yml]: The README says Amazon Linux 2 is also supported - is the epel repo already present in that distro?

Comment (tziebro) [reply mitogen/tasks/main.yml]: amazon linux 2 doesn't need epel for this role. it allows epel to be installed via "amazon-linux-extras install epel"

File: mitogen/tasks/main.yml (hunk 1 above)
Comment (tziebro) [reply mitogen/tasks/main.yml]: Added pip install via meta

File: mitogen/tasks/main.yml (hunk 2 above)
Comment (tziebro) [reply mitogen/tasks/main.yml]: @tequalsme I verified that Amazon Linux 2 includes the python2-pip and python3-pip packages in their core packages while CentOS 7 relies on the EPEL repo for those. I knew there was a reason I only installed EPEL on CentOS 7...I just couldn't remember why.

**Pull request merged**